CHANGELOG.md

```diff
@@ -246,15 +246,15 @@ Note: cURL doesn't follow redirects by default. To avoid breaking cURL and other
 
 The second feature is a face-lift to the directory listing theme and color palette.
 
-> http://bafybeia6po64b6tfqq73lckadrhpihg2oubaxgqaoushquhcek46y3zumm.ipfs.io/
+> http://bafybeia6po64b6tfqq73lckadrhpihg2oubaxgqaoushquhcek46y3zumm.ipfs.localhost:8080
 
 #### IPNS
 
 This release includes several new IPNS and IPNS-related features.
 
 ##### ENS
 
-IPFS now resolves [ENS](https://ens.domains/) names (e.g., [/ipns/ipfs.eth](https://ipfs.eth.dweb.link)) via DNSLink.
+IPFS now resolves [ENS](https://ens.domains/) names (e.g., `/ipns/ipfs.eth`) via DNSLink provided by https://eth.link service.
```

Comment:
`ipfs.eth.dweb.link` is invalid
https://ipfs.eth.ipfs.dweb.link wont have proper TLS setup, better to not link or link to a local gateway instead